### PR TITLE
Feature/activation link

### PR DESCRIPTION
**Pull request by tgreyuk**

### CODE REVIEW

**Comment by tgreyuk** [comment]
[(6) User Accounts - Pending User - Resend Activation Link](https://trello.com/c/HlxXnamI/141-6-user-accounts-pending-user-resend-activation-link)

File: src/app/features/workplace/view-user-account/view-user-account.component.html [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
     <h1 class="govuk-heading-l">{{ user.fullname }}</h1>
   </div>
-  <div class="govuk-grid-column-one-third-from-desktop">
-    <button *ngIf="canDeleteUser" class="govuk-button govuk-button--link" (click)="onDeleteUser()">
-      <img src="/assets/images/bin.svg" /> Delete <span class="govuk-visually-hidden">{{ user.fullname }}'s</span>user
-      account
-    </button>
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <div class="govuk-button-heading-group">
+      <button *ngIf="canResendActivationLink" class="govuk-button govuk-button--link" (click)="resendActivationLink()">
```
Comment (omarCreativeDev) [inline src/app/features/workplace/view-user-account/view-user-account.component.html]: i really like your variable names! this `canResendActivationLink` is super clear

File: src/app/features/workplace/view-user-account/view-user-account.component.ts [hunk 2]
```diff
@@ -127,4 +144,15 @@ export class ViewUserAccountComponent implements OnInit {
       },
     ];
   }
+
+  // TODO users can not delete themselves, but uid for LoggedInSession is not exposed so can't currently compare
+  private setPermissions(users: Array<UserDetails>, auth: LoggedInSession) {
+    const isEdit = auth && auth.role === 'Edit';
```
Comment (omarCreativeDev) [inline src/app/features/workplace/view-user-account/view-user-account.component.ts]: could we use the existing Role enum here?

Comment (tgreyuk) [reply src/app/features/workplace/view-user-account/view-user-account.component.ts]: Yeah ok have implemented that. Just to be picky the enum should be singular really Role.Edit as opposed to Roles.Edit.

File: src/app/features/workplace/view-user-account/view-user-account.component.ts [hunk 3]
```diff
@@ -127,4 +144,15 @@ export class ViewUserAccountComponent implements OnInit {
...
   }
+
+  // TODO users can not delete themselves, but uid for LoggedInSession is not exposed so can't currently compare
+  private setPermissions(users: Array<UserDetails>, auth: LoggedInSession) {
+    const isEdit = auth && auth.role === Roles.Edit;
+    const isPending = this.user.username === null;
+    const isPrimary = this.user.isPrimary;
+    const editUsersList = users.filter(user => user.role === Roles.Edit);
+
+    this.canDeleteUser = isEdit && editUsersList.length > 1 && !isPrimary;
```
Comment (ChazUK) [inline src/app/features/workplace/view-user-account/view-user-account.component.ts]: Do we need to add `this.user.uid !== this.loggedinuser.uid`

Comment (tgreyuk) [reply src/app/features/workplace/view-user-account/view-user-account.component.ts]: Yeah that was a TODO (we are not getting uid in logged in session) but i'll set it up now anyway.

**Pull request merged**